libsolidity/lsp/LanguageServer.cpp

```diff
@@ -171,14 +172,48 @@ void LanguageServer::changeConfiguration(Json::Value const& _settings)
 		if (typeFailureCount)
 			m_client.trace("Invalid JSON configuration passed. \"include-paths\" must be an array of strings.");
 	}
+
+	Json::Value jsonRemappings = _settings["remappings"];
+
+	if (jsonRemappings)
+	{
+		int typeFailureCount = 0;
+		if (jsonRemappings.isArray())
+		{
+			vector<frontend::ImportRemapper::Remapping> remappings;
+			for (Json::Value const& jsonPath: jsonRemappings)
+			{
+				if (jsonPath.isObject())
+				{
+					frontend::ImportRemapper::Remapping remapping = {
+						jsonPath.get("context", "").asString(),
+						jsonPath.get("prefix", "").asString(),
+						jsonPath.get("target", "").asString(),
+					};
+					remappings.emplace_back(remapping);
+				}
+				else
+				{
+					typeFailureCount++;
+				}
+			}
+			m_fileRepository.setRemappings(move(remappings));
+		}
+		else
+			++typeFailureCount;
+
+		if (typeFailureCount)
+			m_client.trace("Invalid JSON configuration passed. \"remappings\" must be an array of objects.");
```

Comment:
I think that's subjective, but maybe it's worth not just mentioning "array of objects" but rather say what that object should be like, especially since it's only three members that it must contain. But than it should be most useful to the reader.